schematic/models/validate_attribute.py

```diff
@@ -378,11 +385,148 @@ def generate_content_error(
             error_row=error_row,
             error_col=attribute_name,
             error_message=error_message,
-            error_val=error_val,
+            error_val=invalid_entry,
         )
 
         return error_list, warning_list
 
+    def _get_rule_attributes(
```

Comment:
This is missing a self argument. (Or it should be a class emthod or a function)

Reply:
GenerateError does not have an init, so its not called for here. If you want though, I can refactor GenerateError to add an init.

Reply:
Oh its another one of these. Don't worry about it for now then. It really doesn't make sense as a class if it has no state, and there's nothing to initialize. Thsi shoudl probably be refactor into util functions.

Reply:
I could make a ticket for a light refactor. I think we could probably put the DMGE an init, or it could be util...

Reply:
Up to you!